packages/flutter_tools/lib/src/android/android_device_discovery.dart

```diff
@@ -61,6 +61,9 @@ class AndroidDevices extends PollingDeviceDiscovery {
   @override
   bool get canListAnything => _androidWorkflow.canListDevices;
 
+  @override
+  bool get supportsWirelessDevices => true;
```

Comment:
Is this PR introducing wireless support for android devices?

Reply:
So Android wireless devices already worked with Flutter. You just had to pair the device through [Android Studio](https://developer.android.com/studio/run/device#wireless) or the command line and then it worked. When you did `flutter devices`, `flutter run`, etc the wireless device would just be grouped with all other devices found.